funnel/models/project.py

```diff
@@ -246,103 +310,165 @@ def datelocation(self):
             daterange = daterange_format.format(
                 start_date=schedule_start_at_date.strftime(strf_date),
                 end_date=schedule_end_at_date.strftime("%d %b"),
-                year=schedule_end_at_date.year)
+                year=schedule_end_at_date.year,
+            )
         return u', '.join(filter(None, [daterange, self.location]))
 
-    schedule_state.add_conditional_state('PAST', schedule_state.PUBLISHED,
-        lambda project: project.schedule_end_at is not None and utcnow() >= project.schedule_end_at,
+    schedule_state.add_conditional_state(
+        'PAST',
+        schedule_state.PUBLISHED,
+        lambda project: project.schedule_end_at is not None
+        and utcnow() >= project.schedule_end_at,
         lambda project: db.func.utcnow() >= project.schedule_end_at,
```

Comment:
having `and` from previous statement and this statement in same indent level is a bit confusing.